concordia/views.py

```diff
@@ -272,7 +272,14 @@ def post(self, request, *args, **kwargs):
 
     def form_valid(self, form):
         self.request.session["captcha_validation_time"] = time()
-        return super().form_valid(form)
+
+        user = form.get_user()
+
+        if user.is_staff:
+            logger.info("Staff tried to log in using the regular form, redirecting")
+            return redirect("/admin/login")
```

Comment:
I think that should have a trailing slash but would probably use reversing just in case we ever relocate this:

```suggestion
            return redirect(reverse("admin:login"))
```